### ISSUE TITLE
Add support for Wingdings and Symbol fonts

### ISSUE THREAD

**Issue by raffaem** [fontist/formulas#70]
I am trying to open a PDF but it says that these fonts are missing:

Symbol, Wingdings, Wingdings 2, Wingdings 3.

Are these available from fontist? 

I also told me that "MT Extra" was missing but I was able to install it

**Comment by ronaldtse** [comment]
Moved to fontist/formula.

**Comment by raffaem** [comment]
Can you also add "Ms PGothic"? https://bigfontsite.com/download/ms-pgothic.html#download

**Comment by alexeymorozov** [comment]
Would appreciate any help with cross-platform unarchiving of RPM packages. The ways I tried so far:

1. [seven_zip_ruby](https://github.com/masamitsu-murase/seven_zip_ruby) should support RPMs, as described [here](https://github.com/masamitsu-murase/seven_zip_ruby/blob/3378f60e4a602a1c9c11a95b09a9e6979f579b98/ext/p7zip/contrib/gzip-like_CLI_wrapper_for_7z/man1/p7zip.1#L14), but it fails on run.
2. [ffi-libarchive](https://github.com/chef/ffi-libarchive) works at least on MacOS, but then it needs to install libarchive on Windows (from Chef or precompiled with fontist).
3. [arr-pm](https://github.com/jordansissel/ruby-arr-pm) works on MacOS, but fails on Windows, since it uses the CLI `xz` utility.
4. [rupert](https://github.com/stefanozanella/rupert) could read filenames on all platforms, but seems it has no unarchiving feature.

**Comment by ronaldtse** [comment]
@alexeymorozov it seems that rupert does have some method for data reading?

https://github.com/stefanozanella/rupert/blob/f4d88c0d1d2847d6a10a00592406bce440b1ffc0/lib/rupert/rpm/entry.rb#L85-L87

**Comment by ronaldtse** [comment]
@alexeymorozov I found this Source RPM package that can be directly un-tared, without necessary for RPM/CPIO unpacking:

https://download.opensuse.org/repositories/home:/Dead_Mozay/openSUSE_Tumbleweed/src/webcore-fonts-3.0-2.14.src.rpm

```sh
$ tar -zxvf webcore-fonts-3.0-2.14.src.rpm 
x webcore-fonts-3.0.tar.gz
x webcore-fonts.spec
$ tar -zxvf webcore-fonts-3.0.tar.gz 
x webcore-fonts/
x webcore-fonts/fonts/
x webcore-fonts/fonts/couri.ttf
x webcore-fonts/fonts/verdanaz.ttf
x webcore-fonts/fonts/cour.ttf
x webcore-fonts/fonts/arialbd.ttf
x webcore-fonts/fonts/comicbd.ttf
...
```

**Comment by alexeymorozov** [comment]
Thanks, Ronald! The source RPM appeared to be easier to extract from. Though still couldn't finish it.

Indeed `rupert` allows to extract an archive file just by reading content after a header [here](https://github.com/stefanozanella/rupert/blob/v0.0.2/lib/rupert/parser.rb#L24):

```ruby
File.write("archive.cpio.gz", @raw_io.read)
```

The resulting file could be unarchived by GzipReader:

```
require 'zlib'
cpio = Zlib::GzipReader.open(filename)
File.write("archive.cpio", cpio.read)
```

But then we get the CPIO file:

```
$ file archive.cpio
archive.cpio: ASCII cpio archive (SVR4 with no CRC)
```

I've tried to unarchive it with [this library](https://github.com/thumblemonks/cpio/blob/master/cpio.rb) with no success, because it supports only [an old format](https://people.freebsd.org/~kientzle/libarchive/man/cpio.5.txt). The formats differs by magic number: 070707 for old, and 070701 for new.

The next step could be to rewrite the lib to support the new format.

**Comment by ronaldtse** [comment]
@alexeymorozov The webcore-fonts-3.0-2.14.src.rpm file can be extracted using just "tar", so there's no need for CPIO or RPM extraction. See the extraction log above.

**Comment by alexeymorozov** [comment]
I've tried to do it with `TarReader` but it didn't work out. Could it be because `tar` supports extraction from `cpio` by default?

```
This implementation can extract from tar, pax, cpio, zip, jar, ar, xar, rpm, 7-zip, and ISO 9660 cdrom images and can create tar, pax, cpio, ar, zip, 7-zip, and shar archives.
```

**Comment by ronaldtse** [comment]
Oh wow! Didn't know that the tar command supports them... 

> The next step could be to rewrite the lib to support the new format.

Let's create a new repo/gem that parses the new CPIO format (while also the old CPIO format): https://github.com/fontist/cpio/issues/1

**Comment by alexeymorozov** [comment]
Found working implementation of the CPIO unarchiver in unused files of `arr-pm` [here](https://github.com/jordansissel/ruby-arr-pm/blob/master/cpio.rb).

Working draft (with some local files) of unarchiving of RPM is in https://github.com/fontist/fontist/pull/207

**Comment by ronaldtse** [comment]
@raffaem the fonts are now available since #74.

Please run `fontist update` and you can install the fonts like `fontist install symbol` or `fontist install 'wingdings 2'`.

Thanks again for the suggestion!

**Comment by raffaem** [comment]
> @raffaem the fonts are now available since #74 released as fontist v1.8.4.
> 
> Please run:
> 
> ```shell
> $ gem install fontist
> $ fontist update
> ```
> 
> Then you can install the fonts like:
> 
> ```shell
> fontist install symbol
> fontist install 'wingdings 2'
> ```
> 
> Thanks again for the suggestion!

Thank you very much. I am having a very hectic period but I was still able to follow this, it is incredible the amount of effort you put into this.

Thank you so much.

Best regards
